test_reproducibility.py

```diff
@@ -29,19 +37,21 @@ def tcase_csv_names():
     ]
 
 
-def compare_results(gt_dir: Path, test_dir: Path, csv_names: list[str]) -> None:
+def compare_results(gt_dir: Path, test_dir: Path, csv_names: list[str], experiment_step: str) -> None:
     for csv_name in csv_names:
-        gt_df = pd.read_csv(gt_dir / csv_name, index_col=0)
+        gt_df = pd.read_csv(gt_dir / experiment_step / csv_name, index_col=0)
         test_df = pd.read_csv(test_dir / csv_name, index_col=0)
         pd.testing.assert_frame_equal(gt_df, test_df, obj=csv_name)
         print(f"Test passed for {csv_name}")
```

Comment:
(2) That will enhance readability of the output: `print(f"Test passed for {csv_name}")` -> `print(f"Test passed for {experiment_step}, {csv_name}")`

Reply:
![Screenshot 2024-07-14 at 17 27 35](https://github.com/user-attachments/assets/d4c049f6-6c46-4abb-872e-c1c9ced5606f)